zpeclient.go

```diff
@@ -172,11 +175,17 @@ func (c *Cache) parseData(item *v1.AthenzDomain) (roleMappings, error) {
 				action:   actionRegex,
 				effect:   effect,
 			}
-			_, ok := crMap.roleToAssertion[assertion.Role]
-			if !ok {
-				crMap.roleToAssertion[assertion.Role] = []*simpleAssertion{}
+			if *assertion.Effect == zms.DENY {
```

Comment:
if `assertion.Effect` is nil would this line panic?

Reply:
assertion effect can only be ALLOW or DENY according to this code: https://github.com/yahoo/athenz/blob/master/clients/go/zms/model.go#L1577-L1580